x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrail.java

```diff
@@ -1154,6 +1173,34 @@ LogEntryBuilder withRequestBody(DeletePrivilegesRequest deletePrivilegesRequest)
             return this;
         }
 
+        LogEntryBuilder withRequestBody(CreateServiceAccountTokenRequest createServiceAccountTokenRequest) throws IOException {
+            logEntry.with(EVENT_ACTION_FIELD_NAME, "create_service_account_token");
+            XContentBuilder builder = JsonXContent.contentBuilder().humanReadable(true);
+            builder.startObject()
+                .startObject("service_account_token")
+                .field("namespace", createServiceAccountTokenRequest.getNamespace())
+                .field("service", createServiceAccountTokenRequest.getServiceName())
+                .field("token_name", createServiceAccountTokenRequest.getTokenName())
```

Comment:
Maybe go with only `name`. It's clear from the context whose name it is, and it's the pattern we use elsewhere too.